README.md

```diff
@@ -22,46 +29,10 @@ For discussions of this library see:
 
 - [Discourse for discussion of scope](https://discourse.bemanproject.org/t/scope-library/315)
 
-# Prior And Other Work
```

Comment:
This information was just moved to lower in the file to emphasize the actual library status

Reply:
We should have better examples of how to do documentations. This looks like a good idea.